activesupport/lib/active_support/testing/parallelize_executor.rb

```diff
@@ -27,6 +27,10 @@ def shutdown
       end
 
       private
+        def parallel_executor
```

Comment:
Making creating the executor lazy so that we don't instantiate anything when parallelization disabled